frontend/src/metabase/query_builder/components/AggregationPopover/AggregationPopover.jsx

```diff
@@ -147,9 +147,9 @@ export default class AggregationPopover extends Component {
     const { aggregationOperators, query, dimension, showRawData } = this.props;
     return (
       aggregationOperators ||
-      (dimension && dimension.aggregationOperators()) ||
+      dimension?.aggregationOperators() ||
       query.table().aggregationOperators()
-    ).filter(agg => showRawData || agg.short !== "rows");
+    ).filter(aggregation => showRawData || aggregation.short !== "rows");
```

Comment:
I'm not sure, but I think `aggregation` and `aggregationOperator` would have different meanings.

I think this is the `aggregationOperator` that this code refers to

https://github.com/metabase/metabase/blob/ea1d0bda76d126904e3b6ee456d0de64a003f4ef/frontend/src/metabase/lib/schema_metadata.js#L599-L608

Reply:
Awesome, thank you, updated.